test/fixtures/bpmn/model-complete-semantic.bpmn

```diff
@@ -896,6 +935,18 @@
           <dc:Bounds x="357" y="2360" width="89" height="53" />
         </bpmndi:BPMNLabel>
       </bpmndi:BPMNShape>
+      <bpmndi:BPMNShape id="shape_start_event_interrupting_escalation_id" bpmnElement="start_event_interrupting_escalation_id">
```

Comment:
Thank you for the good position of the shape of the events 😃 
But, can you increase the size of the following shapes with this value:
```
      <bpmndi:BPMNShape id="shape_participant_1_id" bpmnElement="participant_1_id" isHorizontal="true">
        <dc:Bounds x="160" y="580" width="3954" height="2170" />
      </bpmndi:BPMNShape>
```
```
      <bpmndi:BPMNShape id="shape_expanded_event_sub_process_with_start_events_id" bpmnElement="expanded_event_sub_process_with_start_events_id" isExpanded="true">
        <dc:Bounds x="240" y="1340" width="410" height="1380" />
      </bpmndi:BPMNShape>
```

Reply:
I saw it was fixed in #812. So, I'll approve this PR :)

Thank you